cmd/find.go

```diff
@@ -98,18 +98,34 @@ func regexMatch(pattern, path string) bool {
 	return matched
 }
 
+func getExitStatus(err error) int {
```

Comment:
Can you add a unit test here? @ebozduman

Reply:
Done.